### PR TITLE
AZP/RELEASE: Add Ubuntu 22.04

### PR DESCRIPTION
**Pull request by Al3xR01**
## What
Add ubuntu 22.04 to the release pipeline.

## How ?
The new Ubuntu image was built with Dockerfile & docker-compose below.

### CODE REVIEW

File: buildlib/az-distro-release.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ jobs:
         ubuntu20_cuda11:
           build_container: ubuntu20_cuda11
           artifact_name: $(POSTFIX)-ubuntu20.04-mofed5-cuda11.deb
+          ubuntu22_cuda11_x86_64:
```
Comment (yosefe) [inline buildlib/az-distro-release.yml:34]: indent seems wrong
other container names do not have arch

**Comment by Al3xR01** [comment]
Depends on https://github.com/openucx/ucx/pull/8823

File: buildlib/az-distro-release.yml [hunk 2]
```diff
@@ -31,6 +31,9 @@ jobs:
         ubuntu20_cuda11:
           build_container: ubuntu20_cuda11
           artifact_name: $(POSTFIX)-ubuntu20.04-mofed5-cuda11.deb
+        ubuntu22_cuda11_x86_64:
```
Comment (yosefe) [inline buildlib/az-distro-release.yml:34]: remove x86?

File: buildlib/azure-pipelines-release.yml [hunk 3]
```diff
@@ -21,6 +21,8 @@ resources:
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu18.04-mofed5-cuda11:3
     - container: ubuntu20_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu20.04-mofed5-cuda11:3
+    - container: ubuntu22_cuda11
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/ubuntu22.04-mofed5-cuda11:3
```
Comment (yosefe) [inline buildlib/azure-pipelines-release.yml:25]: remove x86?

Comment (Al3xR01) [reply buildlib/azure-pipelines-release.yml:25]: Removed from the container name.
The Harbor path is correct, as the ARM-based image is also ready and uploaded.

File: buildlib/az-distro-release.yml (hunk 2 above)
Comment (yosefe) [reply buildlib/az-distro-release.yml:34]: seems not removed

Comment (Al3xR01) [reply buildlib/az-distro-release.yml]: Oops... Now it is.

**Review by brminich — APPROVED** [review]

**Review by artemry-nv — APPROVED** [review]

**Review by yosefe — APPROVED** [review]
Release pipeline fix and fixing image paths will be in another PR

**Comment by Al3xR01** [comment]
Closing in favor of https://github.com/openucx/ucx/pull/8832

**Pull request closed without merging**